system/flatcar-linux-update-agent/templates/update-agent.yaml

```diff
@@ -1,35 +1,38 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: container-linux-update-agent
+  name: flatcar-linux-update-agent
   namespace: kube-system
 spec:
   updateStrategy:
     type: RollingUpdate
+  selector:
+    matchLabels:
+      app: flatcar-linux-update-agent
   template:
     metadata:
       labels:
-        app: container-linux-update-agent
-        container-linux-update.v1.coreos.com/agent-version: v0.7.0
+        app: flatcar-linux-update-agent
+        agent-version: v0.7.3
       annotations:
-        container-linux-update.v1.coreos.com/agent-version: v0.7.0
+        agent-version: v0.7.3
     spec:
       tolerations:
       - key: "species"
         operator: "Exists"
         effect: "NoSchedule"
       containers:
       - name: update-agent
-        image: quay.io/coreos/container-linux-update-operator:v0.7.0
+        image: quay.io/kinvolk/flatcar-linux-update-operator:v0.7.3
```

Comment:
Maybe this should be a value that can be updated without requiring a a new chart version